pyFV3/initialization/init_utils.py

```diff
@@ -6,7 +6,7 @@
 
 import ndsl.constants as constants
 from ndsl.dsl.typing import Float
-from ndsl.grid.eta import SURFACE_PRESSURE, compute_eta, vertical_coordinate
+from ndsl.grid.eta import SURFACE_PRESSURE, _compute_eta, _vertical_coordinate
```

Comment:
Oh, that's my mistake, I moved those as private functions without knowing they are actually public. We have to undo my change in NDSL

Reply:
Just updated and pushed again